sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -7,14 +7,14 @@
 ```yaml
 package-name: "@azure/ai-text-analytics"
 title: GeneratedClient
-description: Text Analysis Client
+description: TextAnalytics Client
 generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../
 source-code-folder-path: ./src/generated
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/cognitiveservices-Language-2022-04-01-preview/specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.1/TextAnalytics.json
```

Comment:
This was changed to main, but you really shouldn't use mutable branch or tag specs anyway. Use commits like https://raw.githubusercontent.com/Azure/azure-rest-api-specs/0de25e2a040e1257b3a90faea868ad93c3435e48/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.1/TextAnalytics.json (last edit to that file). Commits are immutable.

Reply:
I understand using swagger commits makes the client generation an idempotent operation. However, the library author often needs to re-generate the library against the most recent version of the swagger most likely to bring in needed bug fixes. So, on the other hand, specifying a commit each time I want to regenerate from an updated swagger is extra work.

From my point of view, I am already involved in the swagger reviews and so far I didn't encounter a situation where pinning which version on main was used in the last client re-generation would have added value/improved my library authoring experience.

That being said, I will revisit this later should such a situation occur.